saveAs/index.js

```diff
@@ -6,6 +6,17 @@ const formats = {
     json: require('./formats/json'),
     csv: require('./formats/csv')
 };
+const randomString = len => {
```

Comment:
Please use crypto.randomBytes instead of this

Reply:
Done https://github.com/softwaregroup-bg/ut-port-sql/pull/89/commits/6cebc588637f1a3fe0693048bc6615f9f640ec8a